specification/frontdoor/resource-manager/readme.nodejs.md

```diff
@@ -6,8 +6,8 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   azure-arm: true
-  package-name: azure-arm-frontdoor
-  output-folder: $(node-sdks-folder)/lib/services/frontdoorManagement
+  package-name: azure-arm-frontDoor
```

Comment:
NPM specifies that package names must be all lowercased. Please revert the changes to this file.

Reply:
changes reverted. I think I only need the changes for c#